src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java

```diff
@@ -115,15 +183,24 @@ public static void setPhysicalInfoSNDcodes(final String lang, final StorageServi
    * @param storageService       the storage service.
    * @param fixedFieldCodesGroup the fixedFieldCodesGroup to populate.
    */
-  public static void setPhysicalInfoRSIcodes(final String lang, final StorageService storageService, final FixedFieldCodesGroup fixedFieldCodesGroup) {
-    fixedFieldCodesGroup.addResults(new FixedFieldElement(SPECIFIC_MATERIAL_DESIGNAION_ON_CODE, storageService.getCodesList(lang, RSI_SPEC_DESIGN).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("altitudeOfSensorCode", storageService.getCodesList(lang, RSI_ALTITUDE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("attitudeOfSensorCode", storageService.getCodesList(lang, RSI_ATTITUDE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("cloudCoverCode", storageService.getCodesList(lang, RSI_CLOUD_COVER).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("platformConstructionTypeCode", storageService.getCodesList(lang, RSI_PLAT_CONSTRUCTION).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("platformUseCode", storageService.getCodesList(lang, RSI_PLAT_USE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("sensorTypeCode", storageService.getCodesList(lang, RSI_SENSOR_TYPE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("remoteSensingDataTypeCode", storageService.getCodesList(lang, RSI_DATA_TYPE).stream().map(toPairItem).collect(toList())));
+  public static void setPhysicalInfoRSIcodes(final String lang, final StorageService storageService,
```

Comment:
AFAICT this is just reformatting... please do this in a separate PR;  or at the **very least** add a comment to the PR description indicating which changes are purely re-formatting.